src/statement.rs

```diff
@@ -55,6 +55,21 @@ impl<'l> Statement<'l> {
         unsafe { ffi::sqlite3_column_count(self.raw.0) as usize }
     }
 
+    /// Return the name of a column in the statement
+    #[inline]
+    pub fn column_name(&self, i: usize) -> &str {
+        unsafe {
+            let ret = ffi::sqlite3_column_name(self.raw.0, i as c_int);
+            c_str_to_str!(ret).unwrap()
+        }
+    }
+
+    /// Return column names in the statement
+    #[inline]
+    pub fn column_names(&self) -> Vec<&str> {
```

Comment:
Do we really need this method? It forces one to collect everything into a vector first and then do what’s needed with the individual elements. What’s your opinion?

Reply:
I thought that the idea of this is as a helper for the user which already checks the boundary.